app-frontend/src/app/components/scenes/sceneFilterPane/sceneFilterPane.scss

```diff
@@ -0,0 +1,16 @@
+@import "../../../../assets/styles/sass/settings/_colors.scss";
+
+.clear-image-source-filter{
```

Comment:
Since @designmatty is working on cleaning up styles, it makes sense to just stick all the random css he's probably going to re-write in the shame file.